programmers/3일차/문자열-다루기-기본.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    const result = parseInt(s);
+
+    if ( (s.length == 4 || s.length == 6) && s == result) {
```

Comment:
```suggestion
    if ( (s.length == 4 || s.length == 6) && s == result) 
```
조건문 한줄이면  괄호 생략 가능합니다